flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.js-meta.xml

```diff
@@ -29,6 +29,7 @@
             <property name="columnsize" type="Integer" label="No. of Columns" default="2" description="Input - This will represent number of columns for View/Edit mode"/>
             <property name="suppressToast" type="Boolean" label="Show Toast on Success?" description="If True, Toast will be triggered on success"/>
             <property name="suppressUnderline" type="Boolean" label="Suppress underlines in View mode?" description="If True, underlines will not be displayed in View mode"/>
+            <property name="boxclass" type="String" label="Main Div Classes" default="slds-box slds-theme_default" description="Main Div Classes"/>
```

Comment:
@mhamzas would probably be beneficial to the user to specify in the `description` attribute that only slds classes are allowed, since we aren't allowing the user to dynamically add custom classes to the css

Reply:
Yes, can change the text but I think it's obvious as the default value has SLDS classes and there is no option to define custom css.

Reply:
Hamzas, thanks for this. What would we add to the documentation to explain how to use it? What are the allowablve values for boxclass?

Reply:
@alexed1 allowable/possible values are defined here: https://www.lightningdesignsystem.com/utilities/themes/
I'll rephrase the Description.
BTW, just realized; Anyone can still use custom css class (Community only) as flow inherits community branding.